src/filter/index.js

```diff
@@ -11,8 +11,12 @@ turndownService.use(turndownPluginGithubFlavouredMarkdown.gfm);
 export const LINKS_TO_CONVERT_SELECTOR = 'a[href]:not([href^="#"])';
 
 export default async function filter(content, { fetch: location, select: extractionSelectors = [], remove: deletionSelectors = [], filter: serviceSpecificFilters = [] }, filterFunctions) {
-  const { document: webPageDOM } = new JSDOM(content).window;
-
+  const virtualConsole = new jsdom.VirtualConsole();
```

Comment:
Why do you need a specific `virtualConsole`?

Reply:
To avoid logging to stdout